tools/Google.Cloud.Tools.SbomGenerator/Program.cs

```diff
@@ -164,23 +164,11 @@ private static Spdx SpdxFrom(string nupkgPath)
     /// </summary>
     /// <param name="nuspecXmlStream">xml content</param>
     /// <returns>metadata</returns>
-    private static Nuspec GetMetadata(Stream nuspecXmlStream)
+    private static NuspecPackage GetMetadata(Stream nuspecXmlStream)
     {
-        var doc = new XmlDocument();
-        var xmlReader = new XmlTextReader(nuspecXmlStream)
-        {
-            Namespaces = false
-        };
-        doc.Load(xmlReader);
-        var metadata = doc.DocumentElement?
-            .SelectSingleNode("/package/metadata");
-        return new Nuspec
-        {
-            Id = metadata?.SelectSingleNode("id")?.InnerText ?? "",
-            Version = metadata?.SelectSingleNode("version")?.InnerText ?? "",
-            Description = metadata?.SelectSingleNode("description")?.InnerText
-                ?? "",
-        };
+        XmlSerializer xmlSerializer = new XmlSerializer(typeof(NuspecPackage));
+
+        return xmlSerializer.Deserialize(nuspecXmlStream) as NuspecPackage;
```

Comment:
I generally prefer casting over `as`, as it has a better failure model. Probably not worth messing with the commit history though...